benchmarks/cpu/arm_mllm_blas_sgemm.cpp

```
@@ -0,0 +1,163 @@
+#include "mllm/mllm.hpp"
+#include "mllm/backends/cpu/kernels/arm/mllm_blas/mllm_blas_sgemm.hpp"
+
+bool is_aligned_16(const void* addr) { return (reinterpret_cast<uintptr_t>(addr) & 0xF) == 0; }
+
+// Optimized for decoding.
+// Q: [1, D]
+// K: [S, D]
+// D is small in mllm's case(small language model).
+// D=64, 96, 128 ...
+static void mllm_blas_matmul_fp32_gemv_nt_t_decode_small_d_qk_baseline(benchmark::State& state) {
+  int D = state.range(0);
```

Clang (14.0.6)

[error] variable 'D' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)